dpnp/dpnp_utils/dpnp_algo_utils.pyx

```diff
@@ -240,6 +240,30 @@ cdef tuple get_shape_dtype(object input_obj):
     return (return_shape, numpy.dtype(type(input_obj)))
 
 
+cdef dparray_shape_type get_common_shape(dparray_shape_type input1_shape, dparray_shape_type input2_shape):
```

Comment:
didn't find where it is used.
Do you expect this algorithm will be the same for all possible variant and functions?
Term "all" - I mean 98% (exceptions might be handled later) of possible variants.

it looks like this is a part of common "broadcast" approach.

Reply:
I expect so.